install.sh

```diff
@@ -298,7 +298,7 @@ echo -e "${RESET}"
 echo_info "Updating crew package information...\n"
 # Without setting LD_LIBRARY_PATH, the mandb postinstall fails
 # from not being able to find the gdbm library.
-export LD_LIBRARY_PATH=$(crew const CREW_LIB_PREFIX | sed -e 's:CREW_LIB_PREFIX=::g')
+export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH=/usr/local/lib$LIB_SUFFIX:/usr/lib$LIB_SUFFIX:/lib$LIB_SUFFIX crew const CREW_LIB_PREFIX | sed -e 's:CREW_LIB_PREFIX=::g')
```

Comment:
I don't think this line should be here.  The default `LD_LIBRARY_PATH` will be either `/usr/local/lib` or `/usr/local/lib64` depending on the architecture.  If something doesn't work then we need to move to the correct default directory.  This is just curing a symptom of the underlying issue.

Reply:
This is only for i686, where the default LD_LIBRARY_PATH doesn't include /usr/local/lib yet when we first run crew, and the wrong libgcc gets used, which kills ruby, since ruby expects a newer libgcc.

Reply:
If not i686, this basically does nothing, since ruby doesn't complain about the older libgcc.